petclinic-frontend/src/features/carts/components/UserCart.tsx

```diff
@@ -8,6 +8,10 @@ interface CartResponseDTO {
   cartId: string;
   customerId: string;
   products: ProductModel[];
+  subtotal: string;
+  tvq: string;
+  tvc: string;
+  total: string;
```

Comment:
Wouldn't be better to use float instead of string since those are currency variables.

Reply:
It is a string, because this way it is easier to control the exact formatting of the price :)